### DISCUSSION TITLE
Update headpost part number

### DISCUSSION THREAD

**Discussion by mekhlakapoor** [AllenNeuralDynamics/aind-metadata-service#179] [General]
# User story
As a user, I want the headpost part number mapping to be updated to indicate the new part numbers. Based on an email from Ali, these are the new mappings and the dates that they were applied:

 Motor ctx headframe: 0160-100-54 - 7/31/23 

Motor ctx well: Currently they are not using a well for the most recent version (-54)

Frontal ctx headframe: 0160-100-56  - 7/17/23

Frontal ctx well: 0160-200-57 - 7/17/23

WHC NP well (for full ring WHC headframe): 0160-073-01 - 7/3/23

# Acceptance criteria
- [ ] These mappings are going to be reliant on the date of the procedure so figure out how we want to handle that

# Sprint Ready Checklist 
- [ ] 1. Acceptance criteria defined
- [ ] 2. Team understands acceptance criteria
- [ ] 3. Team has defined solution / steps to satisfy acceptance criteria
- [ ] 4. Acceptance criteria is verifiable / testable
- [ ] 5. External / 3rd Party dependencies identified
- [ ] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

**Comment by jtyoung84** [comment]
@mekhlakapoor schedule a meeting

**Comment by dyf** [comment]
@mekhlakapoor let's discuss

**Comment by mekhlakapoor** [comment]
If I remember correctly, we brought this up in a meeting but didn't move forward because the part numbers are constantly updating and it would be preferred if this all was tracked somewhere instead of hard-coding. Is this data stored somewhere now that can be fetched?
